lib/code-ribbon-patch.js

```diff
@@ -46,6 +46,29 @@ class CodeRibbonPatch extends Pane {
 
   initialize() {
     this.getElement().classList.add('cr-patch');
+
+    this.onDidAddItem(thing => {
```

Comment:
Where is this command's definition?

Reply:
At this point you'll need to start looking at Atom's source code, specifically our `Patch` is a layer over the Atom `Pane` which I will be forking the element of soon...

If you're for it, you'll want to learn how Panes handle transfers of items (and how they annoyingly don't have references to the sender...) and the bidirectional reference structure of Parent/Children forces us to pass events like this up and down the tree.